src/main/kotlin/org/ozwillo/dcimporter/service/ConnectorsService.kt

```diff
@@ -109,7 +130,20 @@ class ConnectorsService(private val businessAppConfigurationRepository: Business
             }
     }
 
-    fun delete(siret: String, appName: String):Mono<BusinessAppConfiguration>{
+    fun delete(id: String): Mono<BusinessAppConfiguration>{
+        val fallback: Mono<BusinessAppConfiguration> =
+                Mono.error(EmptyException("Connector not found for id: \"$id\""))
+
+        return businessAppConfigurationRepository.findById(id)
```

Comment:
* why loading something you want to delete ?
* why returning something that is deleted ?

Reply:
About the second point : now it return an empty Mono
For the first one : Without making a findById request and check it I really dont know how to handle a request that will have an incorrect id as parameter.

Reply:
yes ok (and it will be updated when integrating the security part)